webpack.sample.config.js

```diff
@@ -52,7 +52,7 @@ let getConfig = function () {
   } catch (err) {
     // create file with defaults if not found
     config = {
-      'LOCAL_OWA_FOLDER': '/home/sims01/openmrs/openmrs-server/owa/',
+      'LOCAL_OWA_FOLDER': '/Users/name/downloads/referenceapplication-standalone-2.6.0/appdata\\owa/',
```

Comment:
Use a path like the one for cohort builder

Reply:
Alright, @dkayiwa. Thank you for the feedback.